developers/wallets/native.md

```diff
@@ -0,0 +1,137 @@
+
+```json
+{
+  "credential_handler": {
+    "url": "/switchboard",
+    "enabledTypes": ["VerifiablePresentation"],
+    "acceptedInput": "url",
+    "acceptedProtocols": [
+      "OID4VCI",
+      "OID4VP",
+      "vcapi"
+    ]
+  }
+}
+```
+
+The `url` property must be the entry point for your mobile wallet so it must be a deep link that will open the application. Details about the event that initiates the opening of this URL will be relayed via query parameters appended to it.
+
+The `"acceptedInput": "url"` line tells CHAPI that the wallet should receive data via opening the URL and providing query parameters rather than browser sent events typically used with Web wallets.
+
+### 2. Allow your users to register their wallet as a Credential Handler with the browser polyfill
+
+Next, we need to have the user trigger the registration of your wallet as a Credential Handler within their browser. This must be initiated by user interaction, so you cannot automatically register without the user clicking a button.
+
+Add something similar to your registration page:
+
+```html
+<button id="installHandlerButton">Register Wallet</button>
+
+<script src="https://unpkg.com/credential-handler-polyfill@3/dist/credential-handler-polyfill.min.js"></script>
+<script src="https://unpkg.com/web-credential-handler@2/dist/web-credential-handler.min.js"></script>
+
+<script>
+  await credentialHandlerPolyfill.loadOnce();
+
+  document.getElementById('installHandlerButton').addEventListener('click', async function() {
+    try {
+      await WebCredentialHandler.installHandler();
+      console.log('Handler Installed Successfully!');
+    } catch (error) {
+      console.error('Error installing handler: ' + error.message);
+    }
+  });
+</script>
+```
+
+### 3. Link users to the registration page from your application
+
+Your mobile application should include a link to your registration page. That link should open the device's default browser and allow them to register the wallet as a Credential Handler.
+
+Once done, your Wallet should now be enabled to handle various credential related requests from across the Web.
+
+## Verifiable Credential Storage
+
+Now that your native wallet has been registered with CHAPI, it can receive requests to store credentials at the URL stated in the `credential_handler.url` of the `manifest.json` above.
+
+The endpoint declared there (i.e. `/switchboard`) will receive a `request` query parameter that will include a URL encoded object with the following properties:
+
+- `credentialRequestOrigin`: This will tell the wallet where the request originated
+- `protocols`: This will include any available credential issuance protocols. The wallet may retrieve the credential for storage from any of the available protocols.
```

Comment:
```suggestion
- `protocols`: This will include any available credential exchange protocols (for issuance and / or presentation). The wallet may retrieve the credential for storage from any of the available protocols.
```